src/Arcus.EventGrid.Publishing/Arcus.EventGrid.Publishing.csproj

```diff
@@ -18,13 +18,17 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
     <PackageReference Include="Arcus.Observability.Telemetry.Core" Version="[2.4.0,3.0.0)" />
+    <PackageReference Include="Arcus.Security.Core" Version="[1.7.0,2.0.0)" />
```

Comment:
Why do we specify upper bounds for the version of these packages ?

Reply:
For breaking changes. This way, we make sure we don't have to worry about invalid compatible packages.

Reply:
I know it is for preventing breaking changes.  However, we do not have a version higher then 3, do we ?  And the build should fail on breaking changes .... 
Of course, when you reference Arcus.EventGrid in a project, and you also reference a higher version of Arcus.Observability then you might have issues.   It's ok for me to leave it here, but imho it is a bit useless for now.

Reply:
Yeah, it's a future-proof setup, you know. We had problems before with the exact situation you described there. A healthy habit of doing it this way will make sure we will never have that problem again in that situation.